go.mod

```diff
@@ -60,9 +58,9 @@ require (
 	github.com/acobaugh/osrelease v0.1.0 // indirect
 	github.com/ajstarks/svgo v0.0.0-20211024235047-1546f124cd8b // indirect
 	github.com/anchore/go-logger v0.0.0-20240217160628-ee28a485904f // indirect
-	github.com/anchore/packageurl-go v0.1.1-0.20240312213626-055233e539b4 // indirect
-	github.com/anchore/stereoscope v0.0.3-0.20240501181043-2e9894674185 // indirect
-	github.com/anchore/syft v1.4.1 // indirect
+	github.com/anchore/packageurl-go v0.1.1-0.20240507183024-848e011fc24f // indirect
+	github.com/anchore/stereoscope v0.0.3 // indirect
+	github.com/anchore/syft v1.13.0 // indirect
```

Comment:
I am not 100% sure that we can upgrade `syft` version, Matthias?

Reply:
@matthyx ?

Reply:
but that upgrade will come with the local SBOM PR

Reply:
ok, I'm OK with that since it's indirect at the moment